src/python/ensembl/io/genomio/genbank/extract_data.py

```diff
@@ -138,16 +134,27 @@ def parse_genbank(self, gb_file):
                 if record.id in organella:
                     record.organelle = organella[record.id]
                 self.seq_records.append(record)
+                if len(self.seq_records) >= 1:
+                    self.format_write_record()
+                else:
+                    logging.warning("No records are found in gb_file")
```

Comment:
This is in a loop over records, so this part should be unindented once, or even twice since we don't need to file anymore at this point.